docs/source/contributing/index.md

```diff
@@ -9,7 +9,6 @@ myst:
 
 # Contributing to `plone.restapi`
 
```

Comment:
This will better introduce this chapter.

````suggestion

We use GNU `make` when developing `plone.restapi`.
To install this package, its dependencies, and its documentation, code formatting, and testing tools, run the following command in the root of the project.

```shell
make
```


````